.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Question 🤔
+about: Usage question or discussion about Metasploit.
+labels: "type: question or discussion"
```

Comment:
This label eneds updated I think https://github.com/rapid7/metasploit-framework/labels?q=question

```suggestion
labels: "question"
```